checks.d/pgbouncer.py

```diff
@@ -0,0 +1,167 @@
+"""Pgbouncer check
+
+Collects metrics from the pgbouncer database.
+"""
+from checks import AgentCheck, CheckException
+from collections import OrderedDict
+
+import psycopg2 as pg
+class ShouldRestartException(Exception): pass
+
+class PgBouncer(AgentCheck):
+    """Collects metrics from pgbouncer
+    """
+    SOURCE_TYPE_NAME = 'pgbouncer'
+    RATE = AgentCheck.rate
+    GAUGE = AgentCheck.gauge
+
+    STATS_METRICS = {
+        'descriptors': [
+            ('database', 'db'),
+        ],
+        'metrics': OrderedDict([
+            ('total_requests', ('pgbouncer.stats.total_requests', RATE)),
```

Comment:
I propose we don't port the name `total_*` straight into our metric names since we change the metric type from `counter` to `rate`.

`total_requests` -> `request_per_s`